dali2mqtt/lamp.py

```diff
@@ -90,6 +90,15 @@ def level(self):
     @level.setter
     def level(self, value):
         """Commit level to ballast."""
+
+        if not isinstance(value, (int, float)):
+            if hasattr(self, '__level'):
```

Comment:
lets keep this simple, not a number ? we return without setting any value

Reply:
We need set value, otherwise you program will crash later

Reply:
can you provide a traceback of later ?

Reply:
You are accessing the value (maybe HA is asking state) and as it is not set the attribute doesn't exists. So first time there need be some value set. But we don't want overwritte existing values if they were set already.